examples/DotNetCore/ConsoleApplicationWithMap/ConsoleApplicationWithMap/Program.cs

```diff
@@ -0,0 +1,158 @@
+            await blobClient.DownloadToAsync(contentStream);
+
+            // Go to the beginning of the content stream
+            contentStream.Position = 0;
+
+            // Convert content to json format
+            var reader = new StreamReader(contentStream);
+            var jsonReader = new JsonTextReader(reader);
+
+            var serializer = new JsonSerializer();
+
+            // Save the string values in the JSON array "Data" to MyBlobContent.Data
+            return serializer.Deserialize<MyBlobContent>(jsonReader);
+        }
+
+        private static void Configure()
+        {
+            var builder = new ConfigurationBuilder();
+
+            // Load a subset of the application's configuration from a json file and environment variables
+            builder.AddJsonFile("appsettings.json")
+                   .AddEnvironmentVariables();
+
+            IConfiguration configuration = builder.Build();
+
+            if (string.IsNullOrEmpty(configuration["ConnectionString"]))
+            {
+                Console.WriteLine("Connection string not found.");
+                Console.WriteLine("Please set the 'ConnectionString' environment variable to a valid Azure App Configuration connection string and re-run this example.");
+                return;
+            }
+
+            // Augment the configuration builder with Azure App Configuration
+            // Pull the connection string from an environment variable
+            builder.AddAzureAppConfiguration(options =>
+            {
+                options.Connect(configuration["ConnectionString"])
+                        .Select("*")
+                        .ConfigureRefresh(refresh =>
+                        {
+                            // key: BlobUri
+                            // value: https://{account_name}.blob.core.windows.net/{container_name}/{blob_name}
+                            refresh.Register("BlobUri")
+                                    .SetCacheExpiration(TimeSpan.FromSeconds(10));
+                        })
+                        .Map(async (setting) =>
+                        {
+                            if (setting.ContentType.Equals("application/storage.blob"))
+                            {
+                                MyBlobContent? blobContent = await ReadBlobContentAsync(new Uri(setting.Value));
+                                string newSettingValue = "";
+
+                                if (blobContent != null)
+                                {
+                                    newSettingValue = $"[{string.Join(", ", blobContent.Data)}]";
+                                }
+                                else
+                                {
+                                    newSettingValue = "Error parsing Blob content.";
+                                }
+
+                                setting = new ConfigurationSetting(setting.Key, newSettingValue, setting.Label, setting.ETag);
```

Comment:
If we dont rely on the etag of mapped ConfigurationSetting, its better to leave it out of the sample.